cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/ReadThroughBookCache.java

```diff
@@ -82,16 +86,20 @@ public BookInfo loadBook (BookId bookId) throws IOException {
         try {
             BookEntity bookEntity = ops.getCradleBookOperator().get(bookId.getName(), readAttrs);
 
+            if (bookEntity == null) {
+                throw new IOException();
```

Comment:
Silent exceptions don't help. Provide a message in exception being thrown. E.g. "No book named '"+bookId.getName()+"' is in database".
And this is not `IOException`, it is `CradleStorageException`, like in `CradleOperators`.